jekyll/_cci2/server/installation/phase-4-post-installation.adoc

```diff
@@ -0,0 +1,533 @@
++
+After running this command, you should see a file named `credentials-velero` in your local working directory.
+
+** **Option 2: Workload Identities**
++
+If you are already using https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity[Workload Identities] in your cluster, you can bind the GCP Service Account you just created to Velero's Kubernetes service account. In this case, the GCP Service Account needs the
+`iam.serviceAccounts.signBlob` role in addition to the permissions already specified above.
+
+. **Install and start Velero**
++
+Run one of the following `velero install` commands, depending on how you authorized the service account. This creates a namespace called `velero` and installs all the necessary resources to run Velero.
++
+NOTE: Backups require https://restic.net/[restic] to operate. When installing Velero, ensure that you have the `--use-restic` flag set.
+
+** **using a JSON key file**
++
+[source, bash]
+----
+velero install \
+    --provider gcp \
+    --plugins velero/velero-plugin-for-gcp:v1.2.0 \
+    --bucket $BUCKET \
+    --secret-file ./credentials-velero \
+    --use-restic \
+    --wait
+----
+
+** **using Workload Identities**
++
+[source,bash]
+----
+velero install \
+    --provider gcp \
+    --plugins velero/velero-plugin-for-gcp:v1.2.0 \
+    --bucket $BUCKET \
+    --no-secret \
+    --sa-annotations iam.gke.io/gcp-service-account=$SERVICE_ACCOUNT_EMAIL \
+    --backup-location-config serviceAccount=$SERVICE_ACCOUNT_EMAIL \
+    --use-restic \
+    --wait
+----
++
+For more options on customizing your installation, refer to the https://github.com/vmware-tanzu/velero-plugin-for-gcp#install-and-start-velero[Velero documentation].
+
+. **Verify Velero** 
++
+Once Velero is installed on your cluster, check the new `velero` namespace. You should have a Velero deployment and a restic daemonset, for example:
++
+[source,bash]
+----
+$ kubectl get pods --namespace velero
+NAME                      READY   STATUS    RESTARTS   AGE
+restic-5vlww              1/1     Running   0          2m
+restic-94ptv              1/1     Running   0          2m
+restic-ch6m9              1/1     Running   0          2m
+restic-mknws              1/1     Running   0          2m
+velero-68788b675c-dm2s7   1/1     Running   0          2m
+----
++
+TIP: As restic is a daemonset, there should be one pod for each node in your Kubernetes cluster.
+
+endif::env-aws[]
+
+////
+
+* S3-COMPATIBLE SETUP *
+
+////
+
+[#set-up-backup-and-restore-with-s3-compatible-storage]
+=== Set up backup and restore with S3-compatible storage
+
+The following steps assume you are using S3-compatible object storage, but not necessarily AWS S3, for your backups.
+
+These instructions were sourced from the Velero documentation https://velero.io/docs/v1.6/contributions/minio/[here].
+
+. **Configure `mc` client**
++
+To start, configure https://docs.min.io/minio/baremetal/reference/minio-mc.html[`mc`] to connect to your storage provider:
```

Comment:
It's really unclear that the mc in the code block is a link.

Reply:
OK I've put the word configure in the link to make it more visible